### DISCUSSION TITLE
Support for 𝕏 AI Grok

### DISCUSSION THREAD

**Discussion by ReinhardKepplinger** [open-webui/open-webui#5430] [Ideas]
Hello everyone,

I wanted to ask if there are any plans to integrate support for Grok 2 (or future versions) from 𝕏 AI into OpenWebUI. 
Grok offers interesting features, such as advanced text understanding and image generation capabilities, which would make it a strong addition to OpenWebUI.

However, as far as I know, Grok cannot currently perform web searches, which would be another exciting possibility for integration. Especially the planned API from 𝕏 AI could simplify this in the future.
Is this an area that is being considered for future updates?

Thanks for your great work!

**Comment by phobes** [comment]
I second this!

**Comment by spammenotinoz** [comment]
This can be closed, Grok now has an API compatible API, so just add it normally.  It's an excellent LLM

**Comment by tellok** [comment]
<img width="1205" height="175" alt="image" src="https://github.com/user-attachments/assets/8061f2f4-b65b-46cb-a213-0e2762dd2580" />


it's only saying thinking.....
